src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -17,6 +17,8 @@ public partial class PackageDependencyView : UserControl
 
         private WorkspaceModel currentWorkspace;
 
+        private String FeedbackLink = "https://forum.dynamobim.com/t/call-for-feedback-on-dynamo-graph-package-dependency-display/37229";
```

Comment:
@reddyashish I think this should just be:
https://github.com/DynamoDS/Dynamo/issues in general - @QilongTang what do you think? I think your forum post is auto closing in a few days?

Reply:
I have removed the auto closing last night. Also I think owner ship wise I would prefer forum post if you would like to set an example. I understand that team wise we prefer Dynamo issues, maybe we could message that differently